### ISSUE TITLE
CommandBar : no HTML or ARIA cues

### ISSUE THREAD

**Issue by taparikh** [OfficeDev/office-ui-fabric-react#9800]
## Codepen demonstrating the issue
https://developer.microsoft.com/en-us/fabric#/controls/web/commandbar

## Actual behavior:
On running an accessibility check on the command bar fabric component on the Office Fabric UI Website - the result of the Accessibility Test (Custom Widgets -> Cues) reveals that the are no values for HTML or ARIA Cues. There is however a valid role="menubar". Wanted to double-check if this expected behavior or an error. 

## Expected behavior:
The HTML and ARIA cues should have values. 

## Documentation describing expected behavior
PLease find the screenshot below that demonstrates this behavior - 
![html_cues_issue](https://user-images.githubusercontent.com/51679982/61163259-eb030d80-a4c1-11e9-88a1-5990e709f831.JPG)

**Comment by taparikh** [comment]
@Vitalius1  Hi ! Just wanted to follow up on this issue & make sure you have all the information you need to resolve it !

**Comment by Vitalius1** [comment]
@taparikh will take a look later today and come back with an answer. Thanks for the follow up.

**Comment by Vitalius1** [comment]
@taparikh I looked over this issue and I actually consider that the div wrapping the `CommandBar` does not need to have any cues as it's never focused. The first focusable element is actually the first button when navigating with a screen reader. It might be another false positive that Accessibility Insights is flagging but in real world when using a screen reader the behavior is correct.

P.S. to be on the same page, what are the cues that Accessibility Insights is looking for?

**Comment by taparikh** [comment]
@Vitalius1 We tried everything to have some value except none show up for the HTML or Aria cues - but no prop seems to fill those values.

**Comment by micahgodbolt** [comment]
I don't believe accessibility insights is actually giving you an error on that widgets page. The command bar is set up properly (AFAIK) and what you're seeing is that this rule doesn't apply to the control so it's neither red nor green.

**Comment by Vitalius1** [comment]
@micahgodbolt we discussed this issue offline with @taparikh  and indeed the screen reader output is accessible. The author will follow up with details after a detailed confirmation that the command bar is getting the correct output in their app.

**Comment by taparikh** [comment]
@Vitalius1 @micahgodbolt  Can you confirm if this issue is resolved or are you looking into it ?

**Comment by micahgodbolt** [comment]
I believe that there is no issue with the existing accessibility of the command bar and this github issue can be closed.

**Comment by taparikh** [comment]
@micahgodbolt When @Vitalius1  and I tested it on a Teams Call yesterday - the command bar showed no value for HTML or ARIA cues in the accessibility test,. We provided all the aria values as exposed by the component  - however saw no result. Please reply to this comment stating that : Command Bar Component having no values for ARIA & HTML Cues is in fact the intended behavior & this accessibility test can be marked as pass.

**Comment by Vitalius1** [comment]
@taparikh I looked a little bit more to understand what the Accessibility Insights Tool means by cues and looks like this `cues` are not related to the `CommandBar` control because it does not need them. Here is a snippet from the help panel from the tool in regards to cues
![Screen Shot 2019-07-23 at 17 47 14](https://user-images.githubusercontent.com/29514833/61756797-a6555d00-ad72-11e9-983d-ecd012a29b30.png)

…and here is another one:

![Screen Shot 2019-07-23 at 17 51 10](https://user-images.githubusercontent.com/29514833/61756813-b1a88880-ad72-11e9-88c0-b6a0b52f1dbe.png)

So yes, CommandBar does not need to have this values set and it can be marked as pass. Moreover, if you notice when running the assessment on any of our demo pages you will see the same results for the cues section. Having no value when it does not need I believe it’s the correct behavior. Basically you would have values in there only when the control is `disabled, required or readonly`, and even then you will have either an ARIA or a HTML cue but not both because you can set that cue  either with an ARIA cue `aria-disabled` or the HTML counterpart `disabled` and same applies to the other 2 (required and readonly). So again, the issue can be closed and the test marked as pass.

**Comment by taparikh** [comment]
@Vitalius1 @micahgodbolt  Thank you for the detailed explanation! I am closing this issue
